src/systemTest/java/com/flightstats/hub/testcase/WebhookLifecycleTest.java

```diff
@@ -62,25 +64,49 @@ public void setup() {
     @Test
     @SneakyThrows
     public void testWebhookWithNoStartItem() {
-        final String data = "{\"webhook\": \"test1\", \"startitem\":\"false\"}";
+        final String data = "{\"fn\": \"first\", \"ln\":\"last\"}";
 
         createChannel();
-        addWebhook(buildWebhook(EMPTY_STRING));
+        addWebhook(buildWebhook(EMPTY_STRING, 2));
+
+        final List<String> channelItems = addItemsToChannel(data, 10);
+        final List<String> channelItemsPosted = getWebhookCallbackItems(channelItems.size());
 
-        List<String> channelItems = addItemsToChannel(data, 10);
-        verifyWebhookCallback(channelItems);
+        Collections.sort(channelItems);
+        Collections.sort(channelItemsPosted);
+        assertEquals(channelItems, channelItemsPosted);
     }
 
     @Test
     @SneakyThrows
     public void testWebhookWithStartItem() {
-        final String data = "{\"webhook\": \"test2\", \"startitem\":\"true\"}";
+        final String data = "{\"key1\": \"value1\", \"key2\":\"value2\"}";
 
         createChannel();
-        List<String> channelItems = addItemsToChannel(data, 10);
+        final List<String> channelItems = addItemsToChannel(data, 10);
 
-        addWebhook(buildWebhook(channelItems.get(4)));
-        verifyWebhookCallback(channelItems.subList(5, channelItems.size()));
+        addWebhook(buildWebhook(channelItems.get(4), 2));
+        final List<String> channelItemsExpected = channelItems.subList(5, channelItems.size());
+        final List<String> channelItemsPosted = getWebhookCallbackItems(channelItemsExpected.size());
+
+        Collections.sort(channelItemsExpected);
+        Collections.sort(channelItemsPosted);
+        assertEquals(channelItemsExpected, channelItemsPosted);
```

Comment:
I think you can use hamcrest's `assertThat(channelItemsPosted, containsInAnyOrder(channelItemsExpected.toArray()))` here to make this a little more readable.

Reply:
I know. I liked that too. But there is a conflict while adding hamcrest dependency currently. it needs to be excluded from one of the existing dependencies. So I am sticking this simple solution now.

Reply:
Ugh, that is sad. Sounds good.

Reply:
I am not sure that we should be sorting these, generally our webhook assertions should confirm that the order that the callbacks happened is correct and sorting I believe will tamper with that

Reply:
I was able to use Hamcrest (on another branch), I had add it's import before junit in the gradle file, and that worked.
Not sure if that is the same conflict you are seeing now. Based on my comment above I'd prefer a different matcher than containsInAnyOrder though :)